### PR TITLE
Bugfix/restore ability to moderate disallowed comments

### LINKED ISSUE #137: Lost "Edit" and "Remove" commands for comments in the "disallow commenting" mode
Lost "Edit" and "Remove" commands for comments in the "disallow commenting" mode (should be available for admins and moderators)

1. Main mode:
image

2. After "disallow commenting"
image

### PR DESCRIPTION
**Pull request by circus2271**
for user with moderator rights:
localhost_3000_p_4 (2)
localhost_3000_p_4 (3)
then after page refresh
localhost_3000_p_4 (4)

for regular user:
localhost_3000_p_4 (8)
localhost_3000_p_4 (9)

Resolves #137
(sorry for misspellings in admin john's comments)

### CODE REVIEW

File: views/module/comment/cdotauth.pug [hunk 1]
```diff
@@ -37,5 +37,10 @@
             .dotDelimeter ·
             span.cact.remove Удалить
             | {{?}}
+        | {{??it.mod}}
```
Comment (kabalin) [inline views/module/comment/cdotauth.pug:40]: I suggest not to duplicate same actions in a different control statement, but rather refactor existing content of `.cacts` cotnainer:
1. Remove condition at line 29 and wrap "reply" action in it, as 
```
{{?it.reply}}
span.cact.reply Ответить
{{?}}
2. Rewrite conditions for "edit" and "delete" actions as `{{?it.mod || (it.reply && c.can.edit)}}`

After above changes you you will likely have a leading dot delimiter in admin/moderator view when commenting is disabled, just hide it using css (hint: we never use `.dotDelimeter` as first child).

**Review by kabalin — CHANGES_REQUESTED** [review]
Hi @circus2271, thanks a lot for contributing! Some changes are needed, see the comments. Also please mention the bug issue id in the commit message body and ideally squash commits into one.

**Review by kabalin — APPROVED** [review]

**Comment by kabalin** [comment]
Thanks Pavel, congratulations on your first PR being merged to PastVu! 🥳

**Comment by circus2271** [comment]
Thanks

**Comment by kabalin** [comment]
`en` branch automerge confict is resolved at 544139864a81ec2acdbb87ea8a794fb0140b9009

**Pull request merged**